### PR TITLE
API-9774: Introduce support for organization ID in v3.4

### PR DESCRIPTION
**Pull request by kacperf531**

### CODE REVIEW

File: livechat/customer/web/client.py [hunk 1]
```diff
@@ -998,14 +1013,163 @@ def cancel_greeting(self,
...
 
 
 class CustomerWeb34(CustomerWebInterface):
     ''' Customer API version 3.4 class. '''
+    def __init__(self, organization_id: str, access_token: str, version: str,
+                 url: str, http2: bool) -> CustomerWeb34:
+        super().__init__(access_token, version, url, http2)
+        if all([organization_id, isinstance(organization_id, str)]):
+            self.organization_id = organization_id
+            self.query_string = f'?organization_id={str(organization_id)}'
```
Comment (skamieniarz) [inline livechat/customer/web/client.py:1106]: Is is necessary to convert it to string (`str()`) considering previous `isinstance(organization_id, str)`?

File: livechat/customer/web/client.py [hunk 2]
```diff
@@ -998,14 +1013,163 @@ def cancel_greeting(self,
...
+        if payload is None:
+            payload = {}
+        params = {}
+        if namespace:
+            params['namespace'] = namespace
+        if name:
+            params['name'] = name
+        if group_id:
+            params['id'] = str(group_id)
+        params['organization_id'] = self.organization_id
```
Comment (skamieniarz) [inline livechat/customer/web/client.py:1171]: Is it the only differing item? Perhaps we could check `dir()` or `vars()` of the client object to check presence of either `license_id` or `organization_id` and apply the one present not to duplicate the definitions. Just a suggestion to consider and check.

**Review by skamieniarz — CHANGES_REQUESTED** [review]

File: livechat/customer/web/client.py [hunk 3]
```diff
@@ -38,25 +41,41 @@ def get_client(license_id: int,
...
         self.api_url = f'https://{base_url}/v{version}/customer/action'
-        self.session = httpx.Client(http2=http2,
-                                    headers={'Authorization': access_token})
-        self.license_id = str(license_id)
+        if all([access_token, isinstance(access_token, str)]):
+            self.session = httpx.Client(
+                http2=http2, headers={'Authorization': access_token})
+        else:
+            raise ValueError(
+                'Incorrect or missing `access_token` argument (should be str.)'
```
[the remark this answers is not in the document]
Comment (kacperf531) [reply livechat/customer/web/client.py:76]: Well, this is something I learned when Ola has suggested in my PR in public docs https://github.com/livechat/livechat-public-docs/pull/1030#pullrequestreview-758414441

File: livechat/customer/web/client.py [hunk 4]
```diff
@@ -998,14 +1013,163 @@ def cancel_greeting(self,
...
     ''' Customer API version 3.3 class. '''
+    def __init__(self, license_id: int, access_token: str, version: str,
+                 url: str, http2: bool) -> CustomerWeb33:
+        super().__init__(access_token, version, url, http2)
+        if all([license_id, isinstance(license_id, int)]):
+            self.license_id = license_id
+            self.query_string = f'?license_id={str(license_id)}'
+        else:
+            raise ValueError(
+                'Incorrect or missing `license_id` argument (should be int.)')
```
[the remark this answers is not in the document]
Comment (kacperf531) [reply livechat/customer/web/client.py:1031]: same as above

File: livechat/customer/rtm/client.py [hunk 5]
```diff
@@ -629,7 +634,29 @@ def update_event_properties(self,
 
 class CustomerRTM33(CustomerRTMInterface):
     ''' Customer RTM version 3.3 class. '''
+    def __init__(self, license_id: int, version: str,
+                 url: str) -> CustomerRTM33:
+        if all([license_id, isinstance(license_id, int)]):
```
[the remark this answers is not in the document]
Comment (kacperf531) [reply livechat/customer/rtm/client.py]: API-9774 post review fixes: 84c537ed54c3cf1d0a1de6cbd63750c2ccc12302

**Review by skamieniarz — DISMISSED** [review]

**Review by skamieniarz — APPROVED** [review]

**Pull request merged**